README.md

```diff
@@ -83,11 +83,8 @@ You've got a local Journaly PostgreSQL DB, woohoo! 🎉
 ### Running Journaly
 
 1. To run the entire app in local development mode, simply run `npm run dev` from the root of the project! This will use `concurrently` to start up both the frontend and backend dev servers.
-2. If you would like to run frontend or backend separately, you can run `npm run dev` from each respective directory.
-
-### Seeding your database
-
-3. Let's seed that baby DB!  
+1. If you would like to run frontend or backend separately, you can run `npm run dev` from each respective directory.
+1. Let's seed that baby DB!  
```

Comment:
Since you're updating these let's update the docs so that they recommend running `npm run reseed-db` from the project root now